src/fidesctl/api/ctl/ui.py

```diff
@@ -0,0 +1,31 @@
+"""Various functions to help serve the UI from our server"""
+
+import importlib.util
+from pathlib import Path
+from typing import Optional
+
+from fastapi import Response
+from fastapi.responses import FileResponse
+
+ADMIN_UI_DIRECTORY = "ui-build/static/admin/"
+
+
+def get_path_to_ui_file(package_name: str, path: str) -> Optional[Path]:
+    """Return a path to a UI file within a given package"""
+    spec = importlib.util.find_spec(package_name)
+    if spec is None:
+        return None
+    return Path(spec.origin).parent / path if spec.origin else None
+
+
+def get_path_to_admin_ui_file(path: str) -> Optional[Path]:
+    """Return a path to an admin UI file."""
+    return get_path_to_ui_file("fidesctl", ADMIN_UI_DIRECTORY + path)
```

Comment:
I'm nervous about just passing `fidesctl` here, since if we ever change the package name, we might forget to do the same here. We could use something like `__name__` or `__package__` instead but that changes the number of `.parent`s we need to use on L18 (`__package__` would return `fidesctl.api.ctl` and `__name__` would return `fidesctl.api.ctl.ui`) and I wanted to keep `get_path_to_ui_file` flexible for fidesctl+. open to other ideas! maybe using `__package__` and just getting the first part of it? 🤔

Reply:
makes sense to me to do something like `__package__.split(".")[0]`